scripts/linters/pre_commit_linter.py

```diff
@@ -32,17 +32,17 @@
 2.  To lint all files in the folder or to lint just a specific file
         python -m scripts.linters.pre_commit_linter --path filepath
 
-3.  To lint a specific list of files (*.js/*.py only). Separate files by spaces
+3.  To lint a specific list of files. Separate files by spaces
```

Comment:
Separate files --> Separate filepaths

and maybe change file_1 to filepath_1, etc.

Reply:
Done.